crates/extensions/tedge_log_manager/src/actor.rs

```diff
@@ -251,23 +231,27 @@ impl LogManagerActor {
         let msg = MqttMessage::new(&self.config.logtype_reload_topic, payload).with_retain();
         self.mqtt_publisher.send(msg).await
     }
+
+    async fn publish_command_status(
+        &mut self,
+        topic: &Topic,
+        request: &LogUploadCmdPayload,
+    ) -> Result<(), ChannelError> {
+        match request_into_message(topic, &request) {
```

Comment:
```suggestion
        match request_into_message(topic, request) {
```
it would be dereferenced by compiler anyway (we pass the reference to this method)

Reply:
fixed